cmd/kubeadm/app/apis/output/types.go

```diff
@@ -82,3 +84,43 @@ type UpgradePlan struct {
 
 	ConfigVersions []ComponentConfigVersionState
 }
+
+// Certificate represents information about check-expiration for one certificate
+type Certificate struct {
+	// Name of the certificate
+	Name string
+	// ExpirationDate defines certificate expiration date.
```

Comment:
```suggestion
	// Name of the certificate.
	Name string
	
	// ExpirationDate defines certificate expiration date.
```

let's separate fields with new line,